tests/integration/components/self-clear-cache-test.js

```diff
@@ -0,0 +1,93 @@
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import { click, render } from '@ember/test-helpers';
+import { hbs } from 'ember-cli-htmlbars';
+
+module('Unit | Component | self-clear-cache', (hooks) => {
+  setupRenderingTest(hooks);
+
+  test('should show last time user cleared the cache', async function (assert) {
```

Comment:
Since these tests currently won't work, let's use `skip` instead of `test`

Please ensure that the develop is aware that they need to make this update when implementing the feature

Reply:
sure